.jenkins/pytorch/test.sh

```diff
@@ -329,6 +329,9 @@ test_libtorch() {
 
 test_aot_compilation() {
   echo "Testing Ahead of Time compilation"
+  ln -sf "$TORCH_LIB_DIR"/libc10* "$TORCH_BIN_DIR"
+  ln -sf "$TORCH_LIB_DIR"/libtorch* "$TORCH_BIN_DIR"
+
```

Comment:
The TEST_REPORTS_DIR wasn't set, that's why we had a failure.

```suggestion

  # Make test_reports directory
  # NB: the ending test_libtorch must match the current function name for the current
  # test reporting process (in print_test_stats.py) to function as expected.
  TEST_REPORTS_DIR=test/test-reports/cpp-unittest/test_aot_compilation
  mkdir -p $TEST_REPORTS_DIR
```